src/Files.App/Filesystem/FilesystemOperations/ShellFilesystemOperations.cs

```diff
@@ -27,6 +27,8 @@ public class ShellFilesystemOperations : IFilesystemOperations
 
 		private FilesystemOperations filesystemOperations;
 
+		private RecycleBinHelpers recycleBinHelpers;
+
 		private IDialogService DialogService { get; } = Ioc.Default.GetRequiredService<IDialogService>();
```

Comment:
This should be refactored

Reply:
It has been removed because it's not used